plugins/inputs/solr/README.md

```diff
@@ -0,0 +1,31 @@
+# Solr input plugin
+
+The [solr](http://lucene.apache.org/solr/) plugin collects stats via the 
+[MBean Request Handler](https://cwiki.apache.org/confluence/display/solr/MBean+Request+Handler)
+
+More about [performance statistics](https://cwiki.apache.org/confluence/display/solr/Performance+Statistics+Reference)
+
+### Configuration:
+
+```
+[[inputs.solr]]
+  ## specify a list of one or more Solr servers
+  servers = ["http://localhost:8983"]
+  ##
+  ## specify a list of one or more Solr cores (default - all)
+  # cores = ["main"]
+```
+
+### Example output of gathered metrics:
+
+```
+➜  ~ telegraf -config telegraf.conf -input-filter solr -test
+* Plugin: solr, Collection 1
+> solr_core,core=main,handler=searcher,host=testhost class_name="org.apache.solr.search.SolrIndexSearcher",deleted_docs=17616645i,max_docs=261848363i,num_docs=244231718i 1478214949000000000
+> solr_core,core=main,handler=core,host=testhost class_name="main",deleted_docs=0i,max_docs=0i,num_docs=0i 1478214949000000000
+> solr_queryhandler,core=main,handler=/replication,host=testhost 15min_rate_reqs_per_second=0.000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000444659081257,5min_rate_reqs_per_second=0.00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000014821969375,75th_pc_request_time=16.484211,95th_pc_request_time=16.484211,999th_pc_request_time=16.484211,99th_pc_request_time=16.484211,avg_requests_per_second=0.0000008443809966322143,avg_time_per_request=12.984811,class_name="org.apache.solr.handler.ReplicationHandler",errors=0i,handler_start=1474662050865i,median_request_time=11.352427,requests=3i,timeouts=0i,total_time=38.954433 1478214949000000000
```

Comment:
This is not a problem with this pull request, but that float format is silly.  Seems like we should switch to using the 'g' fmt with [FormatFloat](https://golang.org/pkg/strconv/#FormatFloat).